src/sharding/Shard.js

```diff
@@ -266,6 +266,8 @@ class Shard extends EventEmitter {
     if (!this.process && !this.worker) return Promise.reject(new Error('SHARDING_NO_CHILD_EXISTS', this.id));
 
     // Cached promise from previous call
+
+    if (typeof script === 'function') script = `(${script})(this)`;
```

Comment:
Keep the comment to its line of code.
```suggestion
    if (typeof script === 'function') script = `(${script})(this)`;

    // Cached promise from previous call
```

Reply:
This is redundant now.
https://github.com/discordjs/discord.js/blob/a5d3dad32a8c6c2fd2148b5ecd9ab54263b6c07b/src/sharding/Shard.js#L285